data/sql_updates/large_aggregates.sql

```diff
@@ -123,7 +123,43 @@ party_totals as (
     group by
         month,
         year
-), -- merge
+),
+communication_totals as (
+  select
+    get_cycle(rpt_yr) as cycle,
+    f.rpt_yr as year,
+    extract(month from to_date(cast(cvg_end_dt as text), 'YYYY-MM-DD')) as month,
+    sum(coalesce(ttl_communication_cost ,0)) as comms_totals
+  from public.fec_vsum_f7_vw f
```

Comment:
I can substitute that view and see if the totals change.  I'm thinking I didn't catch the discussion with @PaulClark2  about why we should use `fitem` versus any of the other tables.  I can substitute that table in and see how the results change.